modules/reference/pages/admin/bootstrap-script.adoc

```diff
@@ -1,52 +1,33 @@
 [[s3-sattools-config-bootstrap]]
 = Bootstrap Script
 
-The menu:Manager Configuration[Bootstrap Script] page allows you to generate a bootstrap script that registers the client systems with {productname} and disconnects them from the remote {scc}.
+In the menu:Manager Configuration[Bootstrap Script] section you can generate a custom bootstrap script.
+Bootstrap scripts are used to register clients with {productname}.
+The generated script will be placed in [path]``/srv/www/htdocs/pub/bootstrap/`` on your {productname} Server.
+
+
```

Comment:
Can we avoid such table markup overhead? I also don't like table layout if there are other possibilities.  I'd prefer definition lists such as:

**Option**
Description (multi-line text)
_Default:_ xyz or `value`

**Option2**
Description2 (multi-line text)
_Default:_ xyz or `value`

Reply:
So this is what we were discussing on email about standards for reference material. If we're abiding by DITA standards, then it should be in table format. I know tables are difficult in adoc (and I hate making them as much as anyone else, especially after the feature matrix nastiness last year), but I think they have better readability over long deflists.

Reply:
Usually readability is worse.  At the first glance, they might look nice... But often there are serious white-space issues (for example, column for "description" too small because there are some insanely long option names).  As long as there is only one sorting critearium (option name) table layout is not the best markup solution.

Reply:
I think I need to find some research on that. Scanning deflists can definitely be harder than scanning tables, especially when we're talking about a series of config options or something.

Reply:
ok, but for now we can go for tables.

Reply:
yep, will let you know what I find.